src/routes/guides/building-a-data-source/+page.svelte

```diff
@@ -50,319 +46,75 @@
   </p>
   <SectionTitle title="What are we building?" />
   <p>
-    In this guide we will build a productizer that provides an API matching the
-    <em>test/ioxio-dataspace-guides/Country/BasicInfo</em> definition that we created in the
-    <A href={GUIDES.BUILD_DATA_DEF.href}>{GUIDES.BUILD_DATA_DEF.title}</A> guide. As the definition is
-    an OpenAPI spec you might want to use se some tool like the Swagger Editor to view it in a more human
-    friendly format.
+    We need to build an integration that provides an API matching one of the available definitions.
+    For references, let's use
+    <A
+      href="https://github.com/ioxio-dataspace/sandbox-definitions/blob/main/src/Meteorology/Weather_v0.1.py"
+    >
+      Meteorology/Weather_v0.1
+    </A>. As the definition is an OpenAPI spec you might want to use some tool like the Swagger
+    Editor to view it in a more human friendly format.
   </p>
   <p>
-    In this case it means we will create an API that accepts a <em>POST</em> request at the path
-    <em>/test/ioxio-dataspace-guides/Country/BasicInfo</em>. It could for example be hosted at
-    <em>https://productizer.example.com/test/ioxio-dataspace-guides/Country/BasicInfo</em>.
+    In this case it means API that accepts a <em>POST</em> request at the path
+    <em>Meteorology/Weather_v0_1</em> needs to be created. It could for example be hosted at
```

Comment:
```suggestion
    <em>/Meteorology/Weather_v0.1</em> needs to be created. It could for example be hosted at
```
Paths usually start with a leading `/`. Also the MAJOR and MINOR numbers are in the path separated by `.`, not `_`.